silta-backup/backup/Dockerfile

```diff
@@ -2,12 +2,12 @@ FROM alpine:3.20
 
 RUN apk add --no-cache \
     bash tzdata coreutils \
-    rsync \
     nodejs
+RUN apk add --no-cache 'rsync>3.4.0'
 
-RUN apk add --no-cache --virtual .build-deps npm \
+RUN apk add --virtual .build-deps npm \
```

Comment:
Why this change, is it related?

Reply:
Ah, forgot to re-add the --no-cache back after trying to fix the weird rsync version issue with apk del commands. Will re-add